debian-downloadrootfs.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-# https://github.com/debuerreotype/docker-debian-artifacts/blob/18cb4d0418be1c80fb19141b69ac2e0600b2d601/buster/Dockerfile
-rootfs_url="https://github.com/debuerreotype/docker-debian-artifacts/raw/18cb4d0418be1c80fb19141b69ac2e0600b2d601/buster/rootfs.tar.xz"
-rootfs_sha256="e7d3a00b8dbf97e6dcb22b00ca9248cc2bf59d1a2107001a3ebf7cfb99542a1a"
+# https://github.com/debuerreotype/docker-debian-artifacts/blob/6032f248d825fd35e8b37037b26dc332e4659c64/bullseye/Dockerfile
+rootfs_url="https://github.com/debuerreotype/docker-debian-artifacts/blob/6032f248d825fd35e8b37037b26dc332e4659c64/bullseye/rootfs.tar.xz"
```

Comment:
You have to use the `/raw/` url instead of the `/blob/` url, because now we are downloading the GitHub page with the button to download the file :)